lcls_naming_tool/taxons/fungible_element_taxon.json

```diff
@@ -0,0 +1,111 @@
+{
+    "4FCCM": "4-F Channel Cut Mono",
+	"4FFOCUS": "Mirror 4F Focus ",
+	"ATM": "Arrival time monitor",
+	"BCS": "Beam containment stopper",
+	"BTM": "Burn Thru Monitor",
+	"BEND": "Bendable mirror, using piezo benders",
+	"C1": "",
+	"C2": "",
+	"CAT1": "",
+	"CAT2": "",
+	"CATCH": "",
+	"COIL": "",
+	"CRYO": "",
+	"DCCM": "Double Channel Cut Mono",
+	"DET": "Detector?",
+	"DG3": "Diagnostic Stand 3",
+	"DIA": "Diagnostic Stand",
+	"DIAG": "Diagnostics",
+	"DIFFRACT": "Diffractometer",
+	"DIFFPUMP": "Diffrenetial Pumping",
+	"DIODE": "Diode",
+	"DP1": "",
+	"DP2": "",
+	"DP3": "",
+	"": "Spectrometer",
+	"EXIT": "Exit slits",
+	"FIM": "Fluorescence Intensity Monitor",
+	"FLAT": "",
+	"GAS": "Attenuator (as opposed to solid)",
+	"GAS_MA_X": "Gas attenuator movable aperture x",
+	"GAS_MA_Y": "Gas attenuator movable aperture y",
+	"GEM": "Gas energy monitor",
+	"GJ": "",
+	"GJ1": "",
+	"GJ2": "",
+	"GMD": "Gas monitor detector",
+	"GSJN": "",
+	"GSJP": "",
+	"HOMS": "Hard-xray offset mirror system, using version I design from Axilon, circa 2017, revised in 2019",
+	"IP1": "Interaction point 1",
+	"IPM": "Intensity and Position Monitor",
+	"K2A": "",
+	"K2B": "",
+	"KBH": "KBO optics (horizontal) (?)",
+	"KBO": "Kirkpatrick-Baez optics",
+	"KBV": "KBO optics (vertical) (?)",
+	"KMONO": "K-line monochromator",
+	"L2SI": "LCLS-2 Strategic Initiative Instruments",
+	"LAS": "General laser prefix",
+	"LD1": "Laser beam protection system - laser destination #1",
+	"LD10": "Laser beam protection system - laser destination #n",
+	"LD14": "Laser beam protection system - laser destination #n",
+	"LD2": "Laser beam protection system - laser destination #n",
+	"LD4": "Laser beam protection system - laser destination #n",
+	"LD6": "Laser beam protection system - laser destination #n",
+	"LD8": "Laser beam protection system - laser destination #n",
+	"LD9": "Laser beam protection system - laser destination #n",
+	"LIC": "Laser in-coupling mirrors",
+	"LOC": "Laser out-coupling mirrors",
+	"LODCM": "Large Offset Double Crystal Monochrometer. Also referred to as LOM (Large Offset Monochrometer)",
+	"LS1": "Laser beam protection system - laser source #n",
+	"LS5": "Laser beam protection system - laser source #n",
+	"LS8": "Laser beam protection system - laser source #n",
+	"LUSI": "LCLS Ultrafast Science Instruments",
+	"MAIN": "",
+	"MC": "",
+	"MECH": "",
+	"MONO": "Monochromater",
+	"NC": "",
+	"PC1K4": "",
+	"PICKER": "Pulse picker",
+	"PLEG": "",
+	"POWER": "Power slits",
+	"PPL": "Patch panel",
+	"PPM": "Power and Profile Monitor",
+	"PPS": "Personel Protection System",
+	"PRE": "",
+	"REF": "Referance Laser",
```

Comment:
Funny typo from the source doc

Reply:
Need to run a spellcheck on the taxon jsons.